platform/administer/templates/parameters.mdx

```diff
@@ -0,0 +1,422 @@
+
+```yaml title="Number parameter with range"
+- variable: replica_count
+  label: Replica Count
+  description: Number of application replicas (1-10)
+  type: number
+  required: true
+  min: 1
+  max: 10
+  default: 3
+  section: Scaling Settings
+```
+
+#### Boolean parameter for feature toggle {#boolean-parameter}
+
+```yaml title="Boolean parameter for feature toggle"
+- variable: enable_monitoring
+  label: Enable Monitoring
+  description: Install Prometheus and Grafana monitoring stack
+  type: boolean
+  default: true
+  section: Optional Features
+```
+
+#### Password parameter for sensitive data {#password-parameter}
+
+```yaml title="Password parameter for sensitive data"
+- variable: admin_password
+  label: Admin Password
+  description: Password for the admin user (minimum 8 characters)
+  type: password
+  required: true
+  validation: "^.{8,}$"
+  section: Security Settings
+```
+
+#### Multiline parameter for configuration {#multiline-parameter}
+
+```yaml title="Multiline parameter for configuration"
+- variable: custom_config
+  label: Custom Configuration
+  description: Additional YAML configuration to apply to the virtual cluster
+  type: multiline
+  section: Advanced Settings
+```
+
+
+### Complex parameter validation
+
+Use advanced validation patterns to restrict and guide user input when filling out template parameters.
+
+#### Enum with descriptions
+
+The following example defines a parameter with a fixed list of allowed options, each with a description to help users choose the correct value. This is useful for selecting cluster sizes, environments, or configuration tiers.
+
+```yaml title="Enum with descriptions (resource tier selection)"
+- variable: instance_size
+  label: Instance Size
+  description: Choose the resource allocation tier for your virtual cluster
+  type: string
+  required: true
+  options:
+    - value: small
+      description: 1 CPU, 2GB RAM - suitable for development
+    - value: medium
+      description: 2 CPU, 4GB RAM - suitable for testing
+    - value: large
+      description: 4 CPU, 8GB RAM - suitable for production workloads
+  section: Resource Configuration
+```
+
+#### Conditional parameters
+
+The following example shows how to conditionally display parameters based on user input. The `storage_size` field only appears if `enable_persistence` is set to `true`. It also uses a regular expression to validate that the input follows a specific storage format.
+
+```yaml title="Conditional parameters with regex validation"
+- variable: enable_persistence
+  label: Enable Persistent Storage
+  description: Add persistent volumes to the virtual cluster
+  type: boolean
+  default: false
+  section: Storage Settings
+
+- variable: storage_size
+  label: Storage Size
+  description: Size of persistent storage (e.g., 10Gi, 50Gi)
+  type: string
+  required: true
+  validation: "^[0-9]+[KMGT]i$"
+  condition: "{{ .Values.enable_persistence }}"
+  section: Storage Settings
+```
```

Comment:
Remove since I don't think `condition` is supported. @kmadel Might know for sure, but I don't see support for this in the backend.